src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -209,10 +195,10 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         operatorIdsToSwap[0] = pubkey.hashG1Point();
 
         // verify the churnApprover's signature
-        _verifyChurnApproverSignatureOnOperatorChurnApproval({
+        _verifyChurnApproverSignature({
             registeringOperatorId: operatorIdsToSwap[0],
             operatorKickParams: operatorKickParams,
-            signatureWithSaltAndExpiry: signatureWithSaltAndExpiry
+            churnApproverSignature: churnApproverSignature
```

Comment:
should we maybe do this earlier in this function?
seems like a fairly late revert condition, but I guess those are somewhat difficult to avoid in this design overall.

Reply:
yes :)

i'm doing this in a future PR